app/main.py

```diff
@@ -1,10 +1,7 @@
 def copy_file(command: str) -> None:
-    find_txt = command.index("txt")
-    first_file_name = command[3 : (find_txt + 3)]
-    second_file_name = command[(find_txt + 4) :]
-
-    with open(first_file_name, "r") as file_in, open(
-        second_file_name, "w"
-    ) as file_out:
-        file_in_content = file_in.read()
-        file_out.write(str(file_in_content))
+    splited_command = command.split()
+    if splited_command[1] != splited_command[2]:
```

Comment:
what if `splited_command == []`?

Reply:
Function take only one argument command, **that is string with command cp, file name to copy and new file name, separated by spaces.**

I thought this parameter isn't empty. 

Please specify, do you mean that I need to add a check like (if splited_command != []?) or my realization isn't correct at all?

Reply:
added checks